src/_utils/item/Item.tsx

```diff
@@ -108,12 +116,23 @@ const Item = ({
       <div className="kirk-item-leftText">
         {leftTitle && (
           <Text
-            className={leftBody ? 'kirk-item-title' : null}
+            className={isLeftTitleInfo ? 'kirk-item-title--withInfo' : null}
```

Comment:
no more `kirk-item-title`?

Reply:
I think it's a legacy rule. There is no logic display link between `leftBody` inside `leftTitle`.
And `kirk-item-title` doesn't apply any styles.